kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/occultism/dimensional_mineshaft/fish.js

```diff
@@ -0,0 +1,61 @@
+onEvent('recipes', (event) => {
+    const id_prefix = 'enigmatica:base/occultism/miners/fish/';
+    const recipes = [
+        {
+            output: 'kubejs:soggy_treasure_box',
+            weight: 177,
+            id: `${id_prefix}soggy_treasure_box`
+        },
+        {
+            output: 'aquaculture:starshell_turtle',
+            weight: 100,
+            id: `${id_prefix}starshell_turtle`
+        },
+        {
+            output: 'aquaculture:arrau_turtle',
+            weight: 100,
+            id: `${id_prefix}arrau_turtle`
+        },
+        {
+            output: 'aquaculture:box_turtle',
+            weight: 100,
+            id: `${id_prefix}box_turtle`
+        },
+        {
+            output: 'aquaculture:leech',
+            weight: 100,
+            id: `${id_prefix}leech`
+        },
+        {
+            output: 'aquaculture:frog',
+            weight: 100,
+            id: `${id_prefix}frog`
+        },
+        {
+            output: 'aquaculture:jellyfish',
+            weight: 100,
+            id: `${id_prefix}jellyfish`
+        },
+        {
+            output: 'aquaculture:fish_bones',
+            weight: 100,
+            id: `${id_prefix}fish_bones`
+        }
+    ];
+
+    fishableFish.forEach((fish) => {
+        recipes.push({
+            output: fish,
+            weight: 100,
+            id: `${id_prefix}${fish.split(':')[0]}`
```

Comment:
I think the index is supposed to be 1 here in order to get the fish name.

Reply:
aha, yep. good catch